src/scenes/Event/scenes/topic.js

```diff
@@ -360,31 +345,32 @@ export default class TopicPage extends React.Component {
     return null;
   };
 
-  renderWinningWithdrawRow = (walletWithinWinnings, index) => {
+  renderWinningWithdrawRow = (withdrawableAddress, index) => {
     const { classes } = this.props;
-    const config = this.getActionButtonConfig(walletWithinWinnings.address);
+    const config = this.getActionButtonConfig(withdrawableAddress.address);
 
     if (!config.show) {
       return null;
     }
 
     return (
-      <TableRow key={walletWithinWinnings.address}>
+      <TableRow key={index}>
```

Comment:
Doesn't using index as key give you a warning?

Reply:
nope. it was actually giving me an error if i had an escrow and winnings withdraw. would only show one.